codegen/smithy-ruby-codegen-test/integration-specs/compression_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
```

Comment:
Can you run rubocop -A on this file?

Reply:
I actually haven't - doing it now, thanks for the call out!